index.html

```diff
@@ -22,17 +22,23 @@ <h1>Olga</h1>
           <figcaption>Olga Mishareva | Web developer</figcaption>
         </figure>
         <div>
-          <h2>Interests</h2>
+          <h3>Skills</h3>
           <ul>
-            <li></li>
-            <li></li>
-            <li></li>
+            <li>HTML</li>
+            <li>CSS</li>
+            <li>JavaScript</li>
+            <li>React</li>
+            <li>Node.js</li>
+            <li>MongoDB</li>
+            <li>Git</li>
           </ul>
-          <h2>Advances</h2>
+          <h3>Interests</h3>
```

Comment:
```suggestion
          <h2>Interests</h2>
```
would suggest again to change the h3 to a h2